packages/react/src/components/Breadcrumbs/BreadcrumbsLink.tsx

```diff
@@ -0,0 +1,27 @@
+import { Slot } from '@radix-ui/react-slot';
+import cl from 'clsx/lite';
+import { forwardRef } from 'react';
+
+import { Link } from '../Link';
+import type { LinkProps } from '../Link';
+
+export type BreadcrumbsLinkProps = LinkProps;
+
+export const BreadcrumbsLink = forwardRef<
+  HTMLAnchorElement,
+  BreadcrumbsLinkProps
+>(({ asChild, className, ...rest }, ref) => {
+  const Component = asChild ? Slot : 'a';
+
+  return (
+    <Link asChild>
```

Comment:
Is this maybe a bit redundant? 

The output component here ends up being the same as if we just used `<Link className={cl("ds-breadcrumbs__link", className)} ref={ref} {...rest}/>`?

Reply:
Agree with @mimarz, take a look at `ListHeading`

Reply:
Agree - was more complex initially so can simplify even more 🤩